Login UseCase/Login.java

```diff
@@ -80,6 +82,8 @@ public boolean authenticate(String username, String password) {
 // LoginController serves as the presentation layer for handling user interaction
 // for login operations. It interacts with the AuthenticationService to manage
 // user login attempts.
+
+
```

Comment:
You Can Return messages instead of printing directly for better UI integration.
You Can add Log authentication attempts.